locale/pt_PT.utf8/LC_MESSAGES/rosariosis.po

```diff
@@ -163,13 +163,13 @@ msgstr "OK"
 
 #: index.php:86
 msgid "Your account has not yet been activated."
-msgstr ""
+msgstr "Sua conta ainda não foi ativada."
 
 #: index.php:86
 #: index.php:151
 #, fuzzy
```

Comment:
Here, the `fuzzy` mention is still present, though you have translated / updated the translation.